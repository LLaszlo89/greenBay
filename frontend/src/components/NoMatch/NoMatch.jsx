<<<<<<< HEAD
import React from 'react';
=======
import React, { useEffect } from 'react';
>>>>>>> new
import { Typography } from '@material-ui/core';

const NoMatch = () => {

return <Typography align="center" ><h1>404 Page Not Found</h1></Typography>;
};

export default NoMatch;
