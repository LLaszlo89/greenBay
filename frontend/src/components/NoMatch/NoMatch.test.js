import React from "react";
import renderer from "react-test-renderer";
import NoMatch from "./NoMatch";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import configureStore from "redux-mock-store";

describe("Connect component to redux store ", () => {
  const mockStore = configureStore([]);

  const store = mockStore( [] );

  it("NoMatch Component should render correctly", () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <NoMatch />
        </Provider>
      </BrowserRouter>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
