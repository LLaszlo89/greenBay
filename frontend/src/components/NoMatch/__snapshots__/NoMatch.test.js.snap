// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect component to redux store  NoMatch Component should render correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter"
>
  <h1>
    404 Page Not Found
  </h1>
</p>
`;
